app/config/collections.php

```diff
@@ -2627,6 +2627,17 @@
                 'array' => false,
                 'filters' => [],
             ],
+            [
+                '$id' => ID::custom('outputSize'),
```

Comment:
Since we are already in the context of a `build` we can simply use `size` to denote this attribute. `Output` is redundant. 

I would also change `outputPath` to just `path` but that's a separate issue

Reply:
Renamed to size and path.